notebooks/Notebook1.md

```diff
@@ -18,17 +20,29 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+One of my big decisions this week was choosing a host language. The three main languages I was considering were scala, R, and Python. I liked the idea for using R because it has a lot of built in things for data and data visualization that would save time, however I also liked the idea of using Python becuase it is a much simpler language than R and still does have good data visualization. The ease of use for the programmer is something really inportant to me, so I decided to not use R. I decided to use Scala as the host language because I have really enjoyed using it so far in this class and wanted to work with it more. I also found that it had lots of data visualization librariesan use to make my graphs. I found this website: https://analyticsindiamag.com/top-7-scala-libraries-for-data-visualisation/ that listed out a bunch of the top ibraries. I think I am going to use EvilPlot mainly because I just like the name. https://github.com/cibotech/evilplot It has teh capability to do line graphs which is all I need for this project. 
+
+Now I want to talk about specific graphs that you would make with my language. All the graphs mainly be would be line graphs. You could use it to track change in weight over time and compare that to amount of calories eaten on each day or the amount of calories burned each day. You could also make a pie chart of the the percentage of each type of food eaten in a given period of time. 
+
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+The most pressuring issue was deciding how  was going to display the graphs. That one decision stemmed for a lot of other design questions like what host language to choose and what stats to track. 
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+What other graphs would you want to make from this data? Would you find this langiage useful? 
```

Comment:
I think it would be great if I could track specific nutrition contents too. Most weight loss/gain apps are about carbs, carbs, carbs - but as a Vegetarian(or anyone in general) being able to track an array of nutrients in my diet would be great. There are so many vitamins and minerals we need - Vitamin A (retinoids and carotenoids)
Vitamin B1 (thiamin)
Vitamin B2 (riboflavin)
Vitamin B3 (niacin)
Vitamin B5 (pantothenic acid)
Vitamin B6 (pyridoxal)
Vitamin B7 (biotin)
Vitamin B9 (folate or folic acid)
Vitamin B12 (cobalamin)
Vitamin C (ascorbic acid)
Vitamin D (cholecalciferol)
Vitamin E (tocopherols and tocotrienols)
Vitamin K (phylloquinone and menaquinones)
alcium
Phosphorus
Magnesium
Sodium
Potassium
Chloride
Sulfur
Iron
Zinc
Copper
Manganese
Iodine
Selenium
Molybdenum
Chromium
Fluoride

It is very hard to keep track of all of these, and more.